### PR TITLE
P14a glider cals

### PR DESCRIPTION
**Pull request by cdobs**
Updates to calibration csvs for instruments deployed on P14a gliders

### CODE REVIEW

File: calibration/CTDGVM/CGINS-CTDGVM-09307__20190623.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9307,CC_calibration_date,1561248000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDGVM/CGINS-CTDGVM-09307__20190623.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00539__20190715.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+539,CC_calibration_date,1563148800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/DOSTAM/CGINS-DOSTAM-00539__20190715.csv:2]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-04093__20190627.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4093,CC_angular_resolution,1.076,
+4093,CC_depolarization_ratio,0.039,
+4093,CC_measurement_wavelength,700,
+4093,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTM/CGINS-FLORTM-04093__20190627.csv:5]: Looks good.

File: calibration/FLORTO/CGINS-FLORTO-01348__20190708.csv [hunk 4]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1348,CC_angular_resolution,1.076,
+1348,CC_depolarization_ratio,0.039,
+1348,CC_measurement_wavelength,700,
+1348,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTO/CGINS-FLORTO-01348__20190708.csv:5]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50218__20190808.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50218,CC_bsipar_par_scaling,1,
```
Comment (snwhite) [inline calibration/PARADM/CGINS-PARADM-50218__20190808.csv:2]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00709__20200611.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
... 272.36, 273.18, 274, 274.82, 275.64, 276.45, 277.27, 278.09, 278.91, 279.73, 280.55, 281.36, 282.18, 283, 283.82, 284.64, 285.46, 286.28, 287.09, 287.91, 288.73, 289.55, 290.37, 291.19, 292, 292.82, 293.64, 294.46, 295.28, 296.1, 296.92, 297.73, 298.55, 299.37, 300.19, 301.01, 301.83, 302.64, 303.46, 304.28, 305.1, 305.92, 306.73, 307.55, 308.37, 309.19, 310.01, 310.82, 311.64, 312.46, 313.28, 314.09, 314.91, 315.73, 316.55, 317.36, 318.18, 319, 319.82, 320.63, 321.45, 322.27, 323.08, 323.9, 324.72, 325.53, 326.35, 327.17, 327.98, 328.8, 329.61, 330.43, 331.25, 332.06, 332.88, 333.69, 334.51, 335.32, 336.14, 336.95, 337.77, 338.58, 339.4, 340.21, 341.03, 341.84, 342.66, 343.47, 344.28, 345.1, 345.91, 346.72, 347.54, 348.35, 349.16, 349.98, 350.79, 351.6, 352.42, 353.23, 354.04, 354.85, 355.66, 356.48, 357.29, 358.1, 358.91, 359.72, 360.53, 361.34, 362.15, 362.96, 363.77, 364.58, 365.39, 366.2, 367.01, 367.82, 368.63, 369.44, 370.25, 371.06, 371.87, 372.67, 373.48, 374.29, 375.1, 375.9, 376.71, 377.52, 378.33, 379.13, 379.94, 380.74, 381.55, 382.36, 383.16, 383.97, 384.77, 385.58, 386.38, 387.18, 387.99, 388.79, 389.6, 390.4, 391.2, 392.01, 392.81, 393.61, 394.41, 395.21, 396.02]",
```
Comment (snwhite) [inline calibration/NUTNRM/CGINS-NUTNRM-00709__20200611.csv:8]: Looks good.

File: calibration/CTDMOG/CGINS-CTDMOG-13419__20200222.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-13419,CC_ptca0,5.253855e+05,
+37-13419,CC_ptca1,9.342711e-01,
+37-13419,CC_ptca2,1.035358e-01,
+37-13419,CC_ptcb0,2.535188e+01,
+37-13419,CC_ptcb1,1.750000e-04,
+37-13419,CC_ptcb2,0.000000e+00,
+37-13419,CC_ptempa0,1.526920e+02,
+37-13419,CC_ptempa1,-7.077361e-02,
+37-13419,CC_ptempa2,7.240583e-07,
+37-13419,CC_wbotc,7.443328e-07,
```
Comment (snwhite) [inline calibration/CTDMOG/CGINS-CTDMOG-13419__20200222.csv:25]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09307__20190623.csv (hunk 1 above)
Comment (rtravis-whoi) [reply calibration/CTDGVM/CGINS-CTDGVM-09307__20190623.csv:2]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00539__20190715.csv (hunk 2 above)
Comment (rtravis-whoi) [reply calibration/DOSTAM/CGINS-DOSTAM-00539__20190715.csv:2]: good

File: calibration/FLORTM/CGINS-FLORTM-04093__20190627.csv (hunk 3 above)
Comment (rtravis-whoi) [reply calibration/FLORTM/CGINS-FLORTM-04093__20190627.csv:5]: good

File: calibration/FLORTO/CGINS-FLORTO-01348__20190708.csv (hunk 4 above)
Comment (rtravis-whoi) [reply calibration/FLORTO/CGINS-FLORTO-01348__20190708.csv:5]: good

File: calibration/NUTNRM/CGINS-NUTNRM-00709__20200611.csv (hunk 6 above)
Comment (rtravis-whoi) [reply calibration/NUTNRM/CGINS-NUTNRM-00709__20200611.csv:8]: good

File: calibration/PARADM/CGINS-PARADM-50218__20190808.csv (hunk 5 above)
Comment (rtravis-whoi) [reply calibration/PARADM/CGINS-PARADM-50218__20190808.csv:2]: good

File: calibration/CTDMOG/CGINS-CTDMOG-13419__20200222.csv (hunk 7 above)
Comment (rtravis-whoi) [reply calibration/CTDMOG/CGINS-CTDMOG-13419__20200222.csv:25]: good - this one is an instrument cal that was already pushed on the cals branch and got lumped in with Collin's glider cals.  Technically Allen created it during QCT so I approve!

**Pull request merged**